Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -999,14 +999,15 @@ function Set-AvsDrsElevationRule {
 
 <#
     .Synopsis
-     Gets all the storage policies available to set on a VM 
+     Gets all the storage policies available to set on a VM.
```

Comment:
If we do filter by Namespace as suggested below, we should say so here and in correspond Set, e.g. these cmdlets support modifying vSAN based storage policies on individual VMs, or similar.